scripts/notify-lark.js

```
@@ -0,0 +1,117 @@
+            }
+        }
+    };
+
+    const res = await Axios.post(chatURL, body, { headers });
+    console.log(res.data);
+}
+
+const args = process.argv.slice(2);
+
+if (args[0]) {
+    sendMessage({
```

ESLint

[error] Delete `··`

(prettier/prettier)